src/Form/form.less

```diff
@@ -240,6 +253,37 @@ base/forms.less
         }
       }
     }
+
+    // Radio button specific styles
+    .form-element-radio-button {
+      input[type="radio"] {
+        & + .form-element-radio-button-decoy {
+          .icon-radio-button {
+            fill: currentColor;
+
+            .large-circle {
+              color: @form-control-default-focus-text-color;
+            }
+
+            .small-circle {
+              color: @white;
+            }
+
+            .shadow {
+              color: @black;
+            }
+          }
+        }
+
+        &:disabled + .form-element-radio-button-decoy {
+          .large-circle {
+            color: #C5C5C5;
+          }
```

Comment:
Let's get a newline above `.large-circle`, and there's an extra newline below this `}`